src/ocm/components/clusterConfiguration/StorageStepTable.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+  Host,
+  OPERATOR_NAME_ODF,
+  isSNO,
+  ClusterHostsTableProps,
+  selectSchedulableMasters,
+} from '../../../common';
+import { numberOfDisks, totalStorageColumn, roleColumn, ODFUsage } from './StorageUtils';
+import { ExpandComponentProps } from '../../../common/components/hosts/AITable';
+import { onDiskRoleType } from '../../../common/components/hosts/DiskRole';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { usePagination } from '../../../common/components/hosts/usePagination';
+import StorageTable from './StorageTable';
+import { AdditionalNTPSourcesDialogToggle } from '../hosts/AdditionaNTPSourceDialogToggle';
+import { HostsTableModals, useHostsTable } from '../hosts/use-hosts-table';
+import { StorageDetail } from './StorageDetail';
+import { countColumn, hostnameColumn } from '../../../common/components/hosts/tableUtils';
+import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
+import { HostsTableEmptyState } from '../../../common/components/hosts/HostsTable';
+import { hardwareStatusColumn } from '../hosts/HostsDiscoveryTable';
+
+const getExpandComponent =
+  (onDiskRole: onDiskRoleType, canEditDisks: (host: Host) => boolean) =>
+  // eslint-disable-next-line react/display-name
+  ({ obj: host }: ExpandComponentProps<Host>) =>
+    (
+      <StorageDetail
+        key={host.id}
+        host={host}
+        onDiskRole={onDiskRole}
+        canEditDisks={canEditDisks}
+        AdditionalNTPSourcesDialogToggleComponent={AdditionalNTPSourcesDialogToggle}
+      />
+    );
+
+const StorageStepTable: React.FC<ClusterHostsTableProps> = ({
+  cluster,
+  setDiscoveryHintModalOpen,
+}) => {
+  const {
+    onEditHost,
+    actionChecks,
+    onDiskRole,
+    actionResolver,
+    onSelect,
```

Comment:
Don't need `onSelect` so it can be removed.
This helps not needing to use `isStorageTable`